ansible/roles_ocp_workloads/ocp4_workload_le_certs_cert_manager/tasks/install-helm.yaml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Check if helm is installed
+  shell: "/usr/bin/which helm  >/dev/null 2>&1"
+  register: is_exist
+  ignore_errors: true
+
+- name: Install helm
+  shell: >
```

Comment:
Why not use ConsoleCLIDownload to find the correct location for Helm and download from there.

We *really* shouldn't be running shell scripts like this.

Worst case save the helm version in the defaults and download from mirror.openshift.com

Reply:
Sure, will get that fixed